webpack.config.js

```diff
@@ -85,6 +86,9 @@ const config = {
   plugins: [
     // Uncomment the following line to enable bundle size analysis
     //    new BundleAnalyzerPlugin()
+    new webpack.ProvidePlugin({
```

Comment:
What is this plugin for?

Reply:
This is the only way I could expose the installed `process`; the below fallback approach didn't work here. This appeared to be the most recommended approach for `process` under v5

Reply:
Do you see a problem during runtime?  Because webpack compiles without it.

Reply:
@stefanv Yes, the page crashes with `process is undefined` without this.

Reply:
I cannot reproduce this on the main page.

Reply:
Meaning this branch runs fine locally without this?

Reply:
Yes, as far as I can tell.

Reply:
@stefanv Unless there's a good reason not to include this fix (fails for me locally without this), I propose we move forward here.

Reply:
Sure; how about adding a comment that this should be removed in the future, and then merge (no need to re-run the CI if it works for you locally).

Reply:
Also for the fallback below.

Reply:
Done, thanks!